AirSkyBoat/AirSkyBoat
# Fixes Mazurka Duration from Harlequin Horn

**_I affirm:_**


## Please enter a player-facing description


Harlequin's Horn now properly increases the duration of Chocobo Mazurka.

## What does this pull request do? (Please be technical)


Updated MAIN_MODIFIER in pTable for Chocobo Mazurka and Raptor Mazurka in `enhancing_song.lua` to use the MAZURKA_EFFECT mod that is added from Harlequin's Horn.

[Wiki Entry](https://ffxiclopedia.fandom.com/wiki/Harlequin's_Horn)


## Steps to test these changes


- `!changejob brd 75`
- `!getmod MAZURKA_EFFECT`
- Cast Chocobo Mazurka and notice duration is set to 120s
- `!additem 17838`
- Equip `Harlequin's Horn`
- `!getmod MAZURKA_EFFECT`
- Cast Chocobo Mazurka and notice duration is now set to 140s (+20s)

## Special Deployment Considerations
